mods/ts/mod.yaml

```diff
@@ -396,3 +396,56 @@ ModContent:
 					^Content/ts/temperat.mix:
 						Offset: 73056940
 						Length: 2037606
+		origin: C&C The Ultimate Collection (Origin version)
```

Comment:
You hard-coded this to the English version which you might want to label as you did above.